system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml

```diff
@@ -55,115 +58,9 @@ spec:
     - name: kubeMonitoring.prometheus.prometheusSpec.thanos.objectStorageConfig.existingSecret.key
       value: thanos.yaml
   {{- end }}
+  {{- if index $serviceValues special }}
+  {{- index $serviceValues special | nindent 4}}
```

Comment:
Why do we name it `special`?

Reply:
renamed to extraSpecs, not sure if we should add `toYaml`

helm template is a bit tricky using secrets with all vault values, any workaround ?

Reply:
for ref
`h template . -f ./ci/test-values.yaml -s templates/kube-monitoring-pluginpreset.yaml`